src/Services/UserService.cs

```diff
@@ -36,4 +42,56 @@ public async Task<User> UpdateUserProfileAsync(Guid userId, UpdateUserDTO user)
         
         return updatedUser;
     }
+
+    public async Task<IEnumerable<MonthlySummaryDTO>> GetMonthlySummaryAsync(Guid userId, DateTime startDate, DateTime endDate)
+    {
+        if (userId == Guid.Empty)
```

Comment:
Apply some desing pattern to your code, is not a good practice use to many if statements in a row

Reply:
This will be applied later together with the FLUENT validation.